### PR TITLE
Feat: Glyphs

### PR DESCRIPTION
**Pull request by dark0dave**
Fix glyphs to use regular charcode.

From
#499
#483

### CODE REVIEW

**Comment by cabarius** [comment]
Can you please implement these as optional with a toggle in the settings tab.  Please make the default be the current ones and define these as "Use Compatibility Glyphs (for Linux, etc)"

**Comment by cabarius** [comment]
Sorry I didn't read this carefully to see that you implemented my request already.  Looking more carefully.

File: ModKit/UI/CharCodeGlyphs.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
+
+namespace ModKit {
+    public static partial class Glyphs {
+        public const string CharCodeCheckOn = "x";
```
Comment (cabarius) [inline ModKit/UI/CharCodeGlyphs.cs:5]: why x and not [x]?

File: ModKit/UI/CharCodeGlyphs.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        public const string CharCodeCheckOff = "o";
+        public const string CharCodeCheckEmpty = "[]";
+        public const string CharCodeDisclosureOn = "v";
+        public const string CharCodeDisclosureOff = ">";
+        public const string CharCodeDisclosureEmpty = "[]";
```
Comment (cabarius) [inline ModKit/UI/CharCodeGlyphs.cs:10]: For disclosure empty please use a space and consider using + and - for v and >

File: ModKit/UI/CharCodeGlyphs.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        public const string CharCodeEdit = "\\";
```
Comment (cabarius) [inline ModKit/UI/CharCodeGlyphs.cs:11]: I am not sure this is clear to a casual user.  Maybe use "Edit"?

**Review by cabarius — CHANGES_REQUESTED** [review]
This looks good in general.  Please consider the UI feedback and after that I'll merge this.  Please forgive my slow response. I am still traveling for another 2 weeks or so.

File: ModKit/UI/CharCodeGlyphs.cs (hunk 2 above)
Comment (dark0dave) [reply ModKit/UI/CharCodeGlyphs.cs]: Done!

File: ModKit/UI/CharCodeGlyphs.cs (hunk 3 above)
Comment (dark0dave) [reply ModKit/UI/CharCodeGlyphs.cs]: Changed to Edit.

**Comment by dark0dave** [comment]
@cabarius no worries. I have updated the pr.

**Review by cabarius — APPROVED** [review]

**Pull request merged**